backend/pkg/api/handlers/common.go

```diff
@@ -337,6 +336,30 @@ func (h *HandlerService) handleDashboardId(ctx context.Context, param string) (*
 	return dashboardId, nil
 }
 
+const chartDatapointLimit uint64 = 200
+
+// helper function to retrieve allowed chart timestamp boundaries according to the users premium perks at the current point in time
+func (h *HandlerService) getCurrentChartTimeLimitsForUser(ctx context.Context, dashboardId *types.VDBId, aggregation enums.ChartAggregation) (uint64, uint64, uint64, error) {
```

Comment:
Minor: IMO when there are this many return values, the values should either be documented in the function doc or be named variables, or maybe you could encapsulate them in a struct.